setup-docker.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+sudo apt install apt-transport-https ca-certificates curl gnupg-agent software-properties-common -y && \
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add - && \
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && \
+sudo apt install docker-ce docker-ce-cli containerd.io -y && \
+sudo apt-mark hold docker-ce && \
+sudo usermod -aG docker $USER && \
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
+sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
It looks same as `scripts/setup-docker.sh` and `docker-compose-collator/docker_setup.sh` above.
I think, 1 version of script is enough.